terraform/main.tf

```diff
@@ -0,0 +1,57 @@
+provider "aws" {
+    region = "eu-west-2"
+}
+
+resource "aws_security_group" "rds_sg"{
+    name = "c14-trend-getter-rds-sg"
+    description = "Security group for trend getter RDS database."
+    vpc_id = var.vpc_id
+
+  ingress {
+    description      = "Allow SSH"
+    from_port        = 22
+    to_port          = 22
+    protocol         = "tcp"
+    cidr_blocks      = ["0.0.0.0/0"] 
```

Comment:
should we be allowing all traffic in?

Reply:
I think we should allow all traffic in for now, to help keep development fast. If we were all on the same private network it would make sense to limit it but we are connecting from different location.

Reply:
Keep in mind for now